src/server/server.ts

```diff
@@ -0,0 +1,26 @@
+import { serve } from "https://deno.land/std/http/server.ts";
```

Comment:
when importing dependencies, use a `deps.ts` file. see the deno-drash repo for an example of how it's done. this is the deno standard

Reply:
good to know